File: API/Entities/Product.cs
namespace API.Entities
{
    public class Product
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string? Description { get; set; }
        public decimal Price { get; set; }
        public string? PictureUrl { get; set; }
        public int CategoryId { get; set; }
        public Category Category { get; set; }
        public string? Ingredients { get; set; }
        public int Inventory { get; set; }
        public string? PublicId { get; set; }

        
    }
}

File: client/src/app/models/product.ts
export interface Product {
    id: number;
    name: string;
    description: string;
    price: number;
    pictureUrl: string;
    category: string;
    ingredients: string;
    inventory: number;
}

export interface ProductLite {
    id: number;
    name: string;
    description: string;
    price: number;
    ingredients: string;
}

export interface ProductParams {
    orderBy: string,
    searchTerm?: string,
    category?: number,
    pageNumber: number,
    pageSize: number,
}